src/napari_imagej/settings.toml

```diff
@@ -0,0 +1,18 @@
+# napari-imagej Settings
+
+# USERS BEWARE:
+# This toml file will soon be replaced with napari's contribution configuration.
+
+# Path to a local ImageJ2 installation (e.g. /Applications/Fiji.app),
+# OR version of net.imagej:imagej artifact to launch (e.g. 2.3.0),
+# OR endpoint of another artifact built on ImageJ2 (e.g. sc.fiji:fiji),
+# OR list of Maven artifacts to include (e.g.
+#   ['net.imagej:imagej:2.3.0', 'net.imagej:imagej-legacy', 'net.preibisch:BigStitcher']).
+# The default is the latest version of ImageJ2.
+imagej_directory_or_endpoint = "net.imagej:imagej"
+
+# This can be used to identify whether transferred data between ImageJ2 and napari
+# should be selected via activation or by user selection via a dialog.
+# By default, the active layer/window is chosen for transfer between applications.
+# By setting this value to false, a popup will be shown instead.
+choose_active_layer = true
```

Comment:
Not really specific to this PR, but... has anyone actually asked for the ability to explicitly select the layer, rather than having it always be the active one? If not, I'd favor removing this feature in favor of always using the active layer. And programmatically, still allowing the layer to be specified in code. Having this option makes testing more complex, and I'm not convinced anyone will want to change this setting.

Reply:
I'm not sure. I feel like @elevans said he liked it.

But, it *does* increase complexity. Do we just want to talk it over in a new issue?

Reply:
I created imagej/napari-imagej#102 to discuss.